src/sentry/ratelimits/utils.py

```diff
@@ -105,6 +106,11 @@ def get_rate_limit_key(
         category = "ip"
         id = ip_address
 
+    # OrgAuthTokens will be treated with IP ratelimits
+    elif is_org_auth_token_auth(request_auth):
+        category = "ip"
+        id = ip_address
```

Comment:
Seems like `ip_address` is always `None` at this point considering the above elif condition:
```python
elif ip_address is not None:
```
Is this intended?

Reply:
actually probably I can remove all of that and just let that be handled by `ip_address is not None` 👍